frontend/src/actionCreators/accessRequestActionCreator.tsx

```diff
@@ -89,12 +95,14 @@ export const getSubmitAccessRequestAction = (accessRequest: IAccessRequest) => (
 export const getSubmitAdminAccessRequestAction = (accessRequest: IAccessRequest) => (
   dispatch: Function,
 ) => {
-  dispatch(request(actionTypes.UPDATE_REQUEST_ACCESS_ADMIN));
```

Comment:
removing this will prevent the isFetching state from being set on the network reducer. This isn't strictly required but is something that all of the other api actions are doing.